rayon-core/src/future/mod.rs

```diff
@@ -184,18 +192,18 @@ impl<'scope, F, S> ScopeFuture<'scope, F, S>
         let spawn = task::spawn(AssertUnwindSafe(future).catch_unwind());
 
         let future: Arc<Self> = Arc::new(ScopeFuture::<F, S> {
-            state: AtomicUsize::new(STATE_PARKED),
-            registry: scope.registry(),
-            contents: Mutex::new(ScopeFutureContents {
-                spawn: None,
-                unpark: None,
-                this: None,
-                scope: Some(scope),
-                waiting_task: None,
-                result: Ok(Async::NotReady),
-                canceled: false,
-            }),
-        });
+                                             state: AtomicUsize::new(STATE_PARKED),
+                                             registry: scope.registry(),
+                                             contents: Mutex::new(ScopeFutureContents {
+                                                                      spawn: None,
+                                                                      unpark: None,
+                                                                      this: None,
+                                                                      scope: Some(scope),
+                                                                      waiting_task: None,
+                                                                      result: Ok(Async::NotReady),
+                                                                      canceled: false,
+                                                                  }),
+                                         });
```

Comment:
(Actually, a better option would to search around for the appropriate conversation in the https://github.com/rust-lang-nursery/fmt-rfcs/ repository...)